### PR TITLE
Fix/TEC-3569 - Fix Eventbrite status filter to not return 'do_not_override' and handle Eventbrite's additional statuses

### PR DESCRIPTION
**Pull request by codingmusician**

### CODE REVIEW

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 1]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
 	public static function filter_setup_do_not_override_post_status( $post_status, $event, $record ) {
-
-		if ( ! isset( $event['eventbrite']->status ) || ! isset( $record->meta['post_status'] ) ) {
-			return $post_status;
-		}
-
-		// We're only interested in the "(do not override)" post status.
-		if ( 'do_not_override' !== $record->meta['post_status'] ) {
-			return $post_status;
+                // override status if set within import
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:160]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 3c3ded45b09aed7017bb50771a649e11</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 2]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                $status = isset( $record->meta['post_status'] ) ? $record->meta['post_status'] : $post_status;
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:161]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: f7c50f38837a34c5910067b3366bc4c3</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 3]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                if ( 'do_not_override' === $status ) {
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:162]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Line indented incorrectly; expected 2 tabs, found 4

<sub>hash: 3e687dc957d4d0ac2b3df32db37ba425</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 4]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                    if( isset($event['eventbrite']->status) ){
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:163]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 5
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 4aa33bdd7c154a62c5386c7270781642</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 5]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                        // draft if set, otherwise publish
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:164]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: c4e434d5a54abf100ccaf4105795bcfb</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 6]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                        if( 'draft' === $event['eventbrite']->status ){
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:165]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Line indented incorrectly; expected 4 tabs, found 6
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 2a3479d0cf604e05bdec84800d65ef67</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 7]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                            $status = 'draft';
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:166]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 5e79e1400f75b7fbb54eb5f96b09f84a</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 8]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                        }else{
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:167]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Line indented incorrectly; expected 4 tabs, found 6
* :no_entry_sign: Expected 1 space after closing brace; 0 found
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after ELSE keyword; 0 found

<sub>hash: 2103955c15ce2fce87cff0847df85b0d</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 9]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                            $status = 'publish';
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:168]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 26a0004866903471543851347effda83</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 10]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                        }
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:169]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Line indented incorrectly; expected 4 tabs, found 6

<sub>hash: 7f6275457a5e05dbfb0be79d972b5d0b</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 11]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                    }else{
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:170]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 5
* :no_entry_sign: Expected 1 space after closing brace; 0 found
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after ELSE keyword; 0 found

<sub>hash: 4a02f01933107f2532a1580caf8d9899</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 12]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                        // if not set, assume publish
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:171]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: d6da2aa391870b290027db483c7ef704</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 13]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                        $status = 'publish';
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:172]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: b4849cac508f7d589aab0704ae20763c</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 14]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                    }
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:173]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 5

<sub>hash: 98ffa119d01b5b0b570ed0b9e609412b</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 15]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                    // If not draft, looked if listed. If not, set to private
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:174]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: ea0710f85ae761368bb68b645805b901</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 16]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                    if( 'draft' !== $status && isset($event['eventbrite']->listed) && !tribe_is_truthy($event['eventbrite']->listed) ){
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:175]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 5
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: Expected 1 space after "!"; 0 found
* :no_entry_sign: ! must be followed by a single space
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 6e614c39354b37364deb70e791301031</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 17]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                        $status = 'private';
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:176]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: d8cb9b18a57abbc345811f640f59e50f</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 18]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+                    }
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:177]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 5

<sub>hash: 15db73c14c931781a54fbeb4a552e427</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 74 error(s)

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 19]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
-			return $post_status;
-		}
-
-		// Currently, 'live' is the only Eventbrite.com status that needs mapping to a WP post status.
-		if ( 'live' === $event['eventbrite']->status ) {
-			return 'publish';
+		// override status if set within import.
+		$status = isset( $record->meta['post_status'] ) ? $record->meta['post_status'] : $post_status;
+		if ( 'do_not_override' === $status ) {
+			if( isset( $event['eventbrite']->status ) ){
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:163]: * :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 290e18fbcf61913f92354e91bf930745</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 20]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+				// draft if set, otherwise publish.
+				if( 'draft' === $event['eventbrite']->status ){
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:165]: * :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: c4acffac21c125f80c9b0764e5b50196</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 21]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+					$status = 'draft';
+				}else{
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:167]: * :no_entry_sign: Expected 1 space after closing brace; 0 found
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after ELSE keyword; 0 found

<sub>hash: 3a54f69426455027934ba8707e14c48a</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 22]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+					$status = 'publish';
+				}
+			}else{
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:170]: * :no_entry_sign: Expected 1 space after closing brace; 0 found
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after ELSE keyword; 0 found

<sub>hash: 9b586a2eeea3c23e78e4a6dd5f271cf5</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 23]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+				// if not set, assume publish.
+				$status = 'publish';
+			}
+			// If not draft, looked if listed. If not, set to private.
+			if( 'draft' !== $status && isset( $event['eventbrite']->listed ) && !tribe_is_truthy( $event['eventbrite']->listed ) ){
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:175]: * :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Expected 1 space after "!"; 0 found
* :no_entry_sign: ! must be followed by a single space
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 0548e07f44dc9a4e2d569bec740cdac6</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 31 error(s)

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 24]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
-			return $post_status;
-		}
-
-		// Currently, 'live' is the only Eventbrite.com status that needs mapping to a WP post status.
-		if ( 'live' === $event['eventbrite']->status ) {
-			return 'publish';
+		// override status if set within import.
+		$status = isset( $record->meta['post_status'] ) ? $record->meta['post_status'] : $post_status;
+		if ( 'do_not_override' === $status ) {
+			if ( isset( $event['eventbrite']->status ) ){
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:163]: * :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: dd8ae1c3533b440893b23b72760a9c57</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 25]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+				// draft if set, otherwise publish.
+				if ( 'draft' === $event['eventbrite']->status ){
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:165]: * :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 7bd7d1fb4b9961683631081ebd71a4a8</sub>

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 26]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+					$status = 'draft';
+				} else {
+					$status = 'publish';
+				}
+			} else {
+				// if not set, assume publish.
+				$status = 'publish';
+			}
+			// If not draft, looked if listed. If not, set to private.
+			if ( 'draft' !== $status && isset( $event['eventbrite']->listed ) && !tribe_is_truthy( $event['eventbrite']->listed ) ){
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:175]: * :no_entry_sign: Expected 1 space after "!"; 0 found
* :no_entry_sign: ! must be followed by a single space
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 8282720e479b428f0fd607b3c023f8a6</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 11 error(s)

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 27]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+					$status = 'draft';
+				} else {
+					$status = 'publish';
+				}
+			} else {
+				// if not set, assume publish.
+				$status = 'publish';
+			}
+			// If not draft, looked if listed. If not, set to private.
+			if ( 'draft' ! == $status && isset( $event['eventbrite']->listed ) && !tribe_is_truthy( $event['eventbrite']->listed ) ) {
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:175]: * :no_entry_sign: PHP syntax error: syntax error, unexpected '!'
* :no_entry_sign: Expected 1 space after "!"; 0 found
* :no_entry_sign: ! must be followed by a single space

<sub>hash: be9ec11c193e4b7bdee79318f8664aef</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 3 error(s)

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 28]
```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
+				// draft if set, otherwise publish.
+				if ( 'draft' === $event['eventbrite']->status ) {
+					$status = 'draft';
+				} else {
+					$status = 'publish';
+				}
+			} else {
+				// if not set, assume publish.
+				$status = 'publish';
+			}
```
Comment (mitogh) [inline src/Tribe/Aggregator/Record/Eventbrite.php:173]: I can recommend to remove the `else` part and make the initial value of the variable instead..

```suggestion
			$status = 'publish';
			if ( isset( $event['eventbrite']->status ) &&  'draft' === $event['eventbrite']->status ) {
				$status = 'draft';
			} 
```

**Review by mitogh — CHANGES_REQUESTED** [review]

**Comment by mitogh** [comment]
Ticket: [TEC-3569]

**Comment by mitogh** [comment]
Would you mind making sure this branch is up to date as well with the branch where is going to be merged `release/B20.11`

File: src/Tribe/Aggregator/Record/Eventbrite.php [hunk 29]
```diff
@@ -157,22 +157,19 @@ public static function filter_add_site_get_import_data( $args, $record ) {
...
-		}
-
-		// Currently, 'live' is the only Eventbrite.com status that needs mapping to a WP post status.
-		if ( 'live' === $event['eventbrite']->status ) {
-			return 'publish';
+		// override status if set within import.
+		$status = isset( $record->meta['post_status'] ) ? $record->meta['post_status'] : $post_status;
+		if ( 'do_not_override' === $status ) {
+			$status = 'publish';
+			if ( isset( $event['eventbrite']->status ) &&  'draft' === $event['eventbrite']->status ) {
```
Comment (tr1b0t) [inline src/Tribe/Aggregator/Record/Eventbrite.php:164]: :no_entry_sign: Expected 1 space after "&&"; 2 found

<sub>hash: 0cb93925109f3ddd017a0c29cc8451e4</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)

**Review by mitogh — APPROVED** [review]

**Comment by codingmusician** [comment]
@mitogh I was thinking that there was a conflict with the readme file, but there wasn't. I reverted the commit, but it had cancelled out your review. Sorry about that.

**Comment by mitogh** [comment]
No worries let me take a look.

**Comment by mitogh** [comment]
Looks like this PR has other changes would you mind making sure those changes are intended and expected? @codingmusician

**Pull request closed without merging**